lib/tasks/auth_source_external.rake

```diff
@@ -0,0 +1,23 @@
+desc 'Create Auth Source External'
```

Comment:
This should also include explanation for what it does and what parameters should be passed to it.